api/log/logger.ts

```diff
@@ -0,0 +1,52 @@
+import winston from 'winston';
+import path from 'path';
+
+const levels = {
+  error: 0,
+  warn: 1,
+  info: 2,
+  http: 3,
+  debug: 4,
+};
+
+const level = () => {
+  const env = process.env.NODE_ENV || 'development';
+  const isDevelopment = env === 'development';
+  return isDevelopment ? 'debug' : 'warn';
+};
+
+const colors = {
```

Comment:
Shouldn't this be uppercased ?

Reply:
Yes, you're right !

Reply:
I"ve also add `as const` after the object declaration to make it `readonly`